platform/platform-app/src/main/resources/features.xml

```diff
@@ -985,6 +985,10 @@
 
     <feature name="platform-all" install="manual" version="${project.version}" hidden="true">
         <feature prerequisite="true">platform-api</feature>
+
+        <!-- This bundle can be removed when Karaf moves to Log4J2 as its default platform -->
```

Comment:
See previous comment.  Include the Karaf upgrade JIRA ticket number here too so it is easy to find later.  Mention to search the code for that ticket.